specification/riskiq/Defender.Easm/main.tsp

```diff
@@ -0,0 +1,1998 @@
+import "@typespec/http";
```

Comment:
It would be better to separate the routes and models into their own files. So having the routes.tsp and models.tsp to make this code easier to go through :)